arpl-uming.txt

```diff
@@ -0,0 +1,18 @@
+[Section]
+Name = AR PL UMing font for ReactOS
+Version = 0.2
+License = the Arphic Public License
+Description = This is a package of AR PL UMing fonts for ReactOS. It is only supported for Chinese (PRC), Chinese (HongKong) and Chinese (Taiwan).This font was developed by Arphic Technology Co., Ltd. in Taiwan.
```

Comment:
Also please add a space after the sentences stop.
"Chinese (Taiwan).This" -> "Chinese (Taiwan). This"